packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -214,6 +214,10 @@ export const SelectedItemCheck = styled( Ariakit.SelectItemCheck )`
 	align-items: center;
 	margin-inline-start: ${ space( 2 ) };
 
+	path {
+		fill: currentColor;
+	}
+
```

Comment:
```suggestion
	fill: currentColor;
```

Isn't this a simpler approach?

Reply:
Thanks for the review, @t-hamano.

Yes, you are right! I've applied this in a1dd30bf3bc559b0da9f5e34d88c5ab27de8d339.

Edit: I was trying to refer to the following snippet https://github.com/WordPress/gutenberg/blob/a1dd30bf3bc559b0da9f5e34d88c5ab27de8d339/packages/components/src/select-control/styles/select-control-styles.ts#L183-L185

But, I believe both should function similarly.